app/components/workflow_table_process_component.rb

```diff
@@ -34,43 +33,44 @@ def rotate_facet_params(prefix, from, to, p = params.dup)
     p[:f] = (p[:f] || {}).dup # the command above is not deep in rails3, !@#$!@#$
     p[:f][from_field] = (p[:f][from_field] || []).dup
     p[:f][to_field] = (p[:f][to_field] || []).dup
-    # rubocop:disable Style/Semicolon
-    p[:f][from_field].reject! { |v| p[:f][to_field] << rotate_facet_value(v, from, to); true }
-    # rubocop:enable Style/Semicolon
+
+    p[:f][from_field].reject! { |v|
```

Comment:
Be OK with multi-line blocks using braces. (I'm meh on this change but can live with it.)

Reply:
I have no idea what it's doing in either case :-)

Reply:
@edsu haha! Yeah, this is rather impenetrable, isn't it?